.tx/config

```diff
@@ -156,3 +156,7 @@ source_file = src/views/microbit/l10n.json
 source_lang = en
 type = KEYVALUEJSON
 
+[scratch-website.parents-l10njson]
```

Comment:
Note: I've added https://github.com/LLK/scratchr2_translations/issues/24 to remember to add the corresponding config in scratchr2_translations.